compiler/z/codegen/OMRLinkage.cpp

```diff
@@ -103,6 +103,37 @@ static int32_t getFirstMaskedBit(int16_t mask); ///< formward reference
 // TR::S390Linkage member functions
 ////////////////////////////////////////////////////////////////////////////////
 
+OMR::Z::Linkage::Linkage(TR::CodeGenerator * codeGen)
+   : OMR::Linkage(),
+      _codeGen(codeGen), _explicitLinkageType(TR_S390LinkageDefault), _linkageType(TR_None), _stackSizeCheckNeeded(true), _raContextSaveNeeded(true),
```

Comment:
I'm debating whether to add a getter/setter for `_linkageType`.  I can't find any use of this field in OMR nor any known downstream project.  If it is deprecated, I'll leave it alone (and open an issue to have it removed).  If there is some use to it then I'll add the getter/setter.  @fjeremic

Reply:
I can't find any uses either. I am doing some work in the area of Linkages on Z right now and I'm not sure if I'll need it. Why don't you open up the issue for deprecating it and assign it to me. Once I get to a PR-ready state on my Linkage work I'll know whether we still need this `_linkageType` exposed or not and if not I'll take care of fixing the said issue.

Reply:
Opened #3717 to track its removal.